hazelcast/src/main/java/com/hazelcast/map/impl/operation/EntryBackupOperation.java

```diff
@@ -123,14 +114,10 @@ private boolean entryRemovedBackup(Map.Entry entry) {
         return false;
     }
 
-    private boolean entryAddedOrUpdatedBackup(Map.Entry entry) {
-        final Object value = entry.getValue();
-        if (value != null) {
```

Comment:
I wrote it before. 

But it is unnecessary because value can not be null there, previous `if` checks in `run method` handles null-ness case. But here is also a fragility because order of `if` checks should not be changed, i added a comment for the importance of `if` check orders for `EntryOperation`, also one similar comment can be added here.